### DISCUSSION TITLE
4/25(화) 의존성 주입

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#103] [General]
의존성 주입이 무엇인지 의존성이 주입을 알아오시면 될 것 같습니다. 앞으로 많이 보게될 단어이기 떄문에 한번쯤은 정리하고 가는 것을 추천드립니다. 시험 기간 얼마 남지 않았지만 최선을 다해 해주기를 바랍니다. 🤨

**Comment by Eunho0922** [comment]
넵
